### PR TITLE
CC release notes 

### LINKED ISSUE #6540: CC: Release Notes for 03/02/2020
PRs to include in the Release Notes:
https://docs.google.com/document/d/19BcvNKCsjrpnSCsOT9slH13dCxTTZGXDqoJlXS6q1HQ/edit?ts=5e42f888

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Closes #6540

### CODE REVIEW

**Comment by kannanlakshmi** [comment]
This LGTM! Yay to getting our first release notes out!!

File: releases/cockroachcloud-02022020.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+toc: true
+summary: Additions and changes in CockroachCloud.
+build_for: [cockroachcloud]
+---
+
+## March 02, 2020
+
+In addition to various updates, enhancements, and bug fixes, this beta release includes the following major highlights:
+
+- For all customers who had opted in for CockroachDB upgrades, CockroachCloud clusters running CockroachDB v19.1 have now been upgraded to [v19.1.8](https://www.cockroachlabs.com/docs/releases/v19.1.8.html) and clusters running CockroachDB v19.2 have been upgraded to [v19.2.4](https://www.cockroachlabs.com/docs/releases/v19.2.4.html). To learn how to opt-in for CockroachDB version upgrades for your CockroachCloud clusters, [contact us](https://support.cockroachlabs.com/hc/en-us).
```
Comment (joshimhoff) [inline releases/cockroachcloud-02022020.md]: For customers running 19.1, we haven't kept them up to date. These are all TF clusters.

For customers running 19.2, we have kept them all up to date (except for a few snowflakes).

**Comment by emily-horing** [comment]
The first bullet should not indicate that minor upgrades are optional. I would change to say something like "the supported minor version of CRDB 19.2 changed from v19.2.2 to v19.2.4. Clusters have been automatically upgraded to the latest supported version and all new clusters are created on v19.2.4"

**Comment by Amruta-Ranade** [comment]
> The first bullet should not indicate that minor upgrades are optional. I would change to say something like "the supported minor version of CRDB 19.2 changed from v19.2.2 to v19.2.4. Clusters have been automatically upgraded to the latest supported version and all new clusters are created on v19.2.4"
> 
> That way you can combine bullets 1 & 2 :)

Hmm..minor/major versions seem internal jargon-y to me. Can we say "Upgraded CockroachCloud clusters running CockroachDB 19.2.2 to v19.2.4. All new clusters will now be created with CockroachDB V19.2.4."

**Comment by emily-horing** [comment]
lgtm

**Comment by Amruta-Ranade** [comment]
Closed via https://github.com/cockroachdb/docs/pull/6718

**Pull request closed without merging**